packages/cli/core/README.md

```diff
@@ -1043,6 +1044,34 @@ EXAMPLES
 
 _See code: [src/commands/ui/create/vue.ts](https://github.com/coveo/cli/blob/@coveo/cli@2.2.3/packages/cli/core/src/commands/ui/create/vue.ts)_
 
+## `coveo ui:deploy`
+
+Deploy your search application to the Coveo infrastructure.
+
+```
+USAGE
+  $ coveo ui:deploy [-p <value>] [-c <value>] [-o <value>]
+
+FLAGS
+  -p, --pageId=7944ff4a-9943-4999-a3f6-3e81a7f6fb0a  The hosted page ID.
+  -c, --config=./configs/myconfig.json [default: coveo.deploy.json] The path to the configuration file.
+  -o, --organization=myOrgID  The identifier of the organization inside which to perform operations. See
+                              <https://docs.coveo.com/en/1562/#organization-id-and-other-information>.
+
+DESCRIPTION
+  Deploy your search application to the Coveo infrastructure.
+
```

Comment:
I think I went a bit too quickly during my first review and didn't understand enough about the config; apologies. Now that I think a bit more about it, I wonder if we expect people to craft the config.json themselves in some cases? If we do, I think we should explain what each param does. We would probably have to do it in this description, something like the following. (I'm not sure how to describe the params)

```suggestion
The `config` must contain the following parameters:

{
  "name": "The name of the hosted search page.",
  "dir": "The directory of the hosted search page.",
  "htmlEntryFile": {
    "path": "The HTML file that ... "
  },
  "javascriptEntryFiles": [
    {
      "path": "The path to ...",
      "isModule": "Whether ... "
    }
  ],
  "javascriptUrls": [
    {
      "path": "https://static.cloud.coveo.com/atomic/v2/atomic.esm.js",
      "isModule": true
    }
  ],
  "cssEntryFiles": [
    {
      "path": "css/index.css"
    }
  ],
  "cssUrls": [
    {
      "path": "https://static.cloud.coveo.com/atomic/v2/themes/coveo.css"
    }
  ]
}

```

Would that make sense or would it just be overkill?

Reply:
Sure we can do that! I'll fill it out